cdap-docs/developer-manual/source/getting-started/start-stop-cdap.rst

```diff
@@ -45,3 +45,13 @@ applications and interact with CDAP.
 Note that in the case of the Docker container running inside a Virtual Machine (as on
 either Mac OS X or Microsoft Windows), you will need to substitute the Docker VM's IP
 address for ``localhost`` in the web browser address bar.
+
+
+Customize JVM OPTS For Sandbox on Linux
+---------------------------------------
+On Linux, you can customize the jvm options for OutOfMemory error and whether to do a heapdump if an OutOfMemory error
+happens. By default, CDAP process will be force killed if an OutOfMemory error happens and there will be no heapdump.
+You can configure these settings by editing the ``cdap-env.sh`` file under your ``conf`` directory. To change
```

Comment:
"To customize these settings, edit the ``cdap-env.sh`` file under the `./conf` directory" (./ added for consistency with `./bin` directory above)